Dcs multi factor authentication (#1682)
Codecademy/docs: content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md

**Multi-factor Authentication (MFA)** is the use of more than one factor to prove the identity of a user.

> One factor is typically a username or email and password combination. This is the basic log-in functionality that most websites require users to create when initially signing up for an account.

Reviewer: Can we create a segue to the other factors below

Author: I am not sure I understand this request. Do you think it pertinent to add another sentence after line 17?

Reviewer: After a second look, I think this is straightforward and good as-is 🙂

## Factors

> There are, broadly speaking, five ways for a user to prove his identity. A user can prove his identity through

Reviewer: ```suggestion
There are, broadly speaking, five ways for a user to prove their identity. A user can prove their identity through the following:
```

Author: This changes have been added.

> - Something the user knows
> - Something the user has
> - Something the user is
> - Some*where* the user is
> - Some*when* the user is

Reviewer: ```suggestion
- Something the user knows.
- Something the user has.
- Something the user is.
- Somewhere the user is.
- Somewhen the user is.
```

Author: I have taken away the italicizing of the text, but I'd rather not add punctuation mark to the list items for the sake of consistency given that we have not added punctuation to the other listed items.

Multi-factor authentication is typically discussed online or in a corporate context as two-factor authentication (2FA) which consists in the user providing two of the above factors as a means to more securely verify his identity. More factors being used in the verification process results in greater security. 

Below are common examples for each of the authentication factors.

## Examples

### Something the User Knows

Typically the first (and often only) factor used to verify user identity. Examples include the user knowing:

- a password
- a personal identification number (PIN)

> - a signature

Author: This changes have been added.

### Something the User Has

This factor refers to a physical object in the possession of the user. Something the user has is often the second factor of authentication referred to in a 2FA authentication procedure. Examples include the user having:

- a mobile phone
- a USB key
- a security token
- a bank card

> - a physical key

Author: These changes have been added.

The physical objects used for verification are signed in such a way as to certify their belonging to the individual whose identity is being verified (e.g., a model number on a lock key, a PIN being sent to the user's phone, etc.)

### Something the User Is

Something the user is typically refers to the user's biology in the form of biometric markers. This factor can also refer to the user's behavioral tendencies. Examples include the user's:

- fingerprint(s)
- face
- voice
- iris(es)

> - keystroke typing dynamics

Author: These changes have been added.

### Somewhere the User Is

This factor uses information about the user's location. Examples include the user's:

- [geolocation](https://www.codecademy.com/resources/docs/general/geolocation)

> - proximity to a particular location

Author: These changes have been added.
